errors.go

```diff
@@ -3,8 +3,25 @@ package servicebus
 import (
 	"fmt"
 	"reflect"
+	"time"
 
 	"github.com/Azure/azure-amqp-common-go/v3/rpc"
+	"github.com/Azure/go-amqp"
+)
+
+// Error Conditions
+const (
+	// Service Bus Errors
+	ErrorServerBusy         amqp.ErrorCondition = "com.microsoft:server-busy"
+	ErrorTimeout            amqp.ErrorCondition = "com.microsoft:timeout"
+	ErrorOperationCancelled amqp.ErrorCondition = "com.microsoft:operation-cancelled"
+	ErrorContainerClose     amqp.ErrorCondition = "com.microsoft:container-close"
```

Comment:
I dont think should/need to expose these error types publicly.

Reply:
Updated.